Editor/AGS.Editor/Panes/Room/RoomSettingsEditor.cs

```diff
@@ -566,19 +569,24 @@ private void ImportBackground(int bgIndex)
                             }
                             else
                             {
-                                roomSizeChanged = true;

```

Comment:
I think this shouldn't have been here because farther down the doImport could be canceled by the dialog box and the room size wouldn't change.  It wouldn't have mattered in practice though since roomSizeChanged only gets used if doImport is true.